sites/all/modules/publizon/publizon.features.field_instance.inc

```diff
@@ -163,6 +163,7 @@ function publizon_field_default_field_instances() {
   t('Confirm your phone number');
   t('Email');
   t('Email (confirm)');
+  t('Enter a Danish 8 digit phone number');
```

Comment:
Not to be a kill-joy, but we have a many library users that many not have an danish number. Could be students etc. how can have a library login. I don't see any number validation in this PR, but if on exists country code should be allowed as part of the number (00xx or +xx).